AppendBlobReader/src/consumer.js

```diff
@@ -677,69 +518,65 @@ function sendDataToSumoUsingSplitHandler(context, dataBytesBuffer, sendOptions,
+        context.log.verbose("Error in appendBlobStreamMessageHandlerv2 Potential throttling scenario: blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
         discardError = true;
     } else if (err !== undefined && (err.code === "ContainerNotFound" || err.code === "BlobNotFound" || err.statusCode == 404)) {
         // sometimes it may happen that the file/container gets deleted
-        context.log("Error in appendBlobStreamMessageHandlerv2 File location doesn't exists:  blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
+        context.log.verbose("Error in appendBlobStreamMessageHandlerv2 File location doesn't exists:  blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
         discardError = true;
     } else if (err !== undefined && (err.code === "ECONNREFUSED" || err.code === "ECONNRESET" || err.code === "ETIMEDOUT")) {
-        context.log("Error in appendBlobStreamMessageHandlerv2 Connection Refused Error: blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
+        context.log.verbose("Error in appendBlobStreamMessageHandlerv2 Connection Refused Error: blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
         discardError = true;
 
     } else {
-        context.log("Error in appendBlobStreamMessageHandlerv2 Unknown error blob %s %d %d %d %s Exit now!", serviceBusTask.rowKey, serviceBusTask.startByte, serviceBusTask.endByte, err.statusCode, err.code);
+        context.log.error(`Error in appendBlobStreamMessageHandlerv2 Unknown error blob rowKey: ${serviceBusTask.rowKey}, startByte: ${serviceBusTask.startByte}, endByte: ${serviceBusTask.endByte}, statusCode: ${err.statusCode}, code: ${err.code} Exit now!`);
     }
     return discardError;
 }
 
-function archiveIngestedFile(serviceBusTask, context) {
-    return tableService.deleteEntity(process.env.APPSETTING_TABLE_NAME, {
-        PartitionKey: serviceBusTask.containerName,
-        RowKey: serviceBusTask.rowKey
-    }, function(err, resp) {
-        if (!err) {
-            context.log("Archived non existing file", serviceBusTask.rowKey);
-        } else {
-            context.log("Error in appendBlobStreamMessageHandlerv2: not able to delete table row", serviceBusTask.rowKey, err);
-        }
-        context.done()
-    });
+async function archiveIngestedFile(serviceBusTask, context) {
+    await azureTableClient.deleteEntity(serviceBusTask.containerName, serviceBusTask.rowKey);
```

Comment:
wrap in in promise and handle error see getData how that was migrated to async

Reply:
okay